### DISCUSSION TITLE
Automatically Delete Backup If Matching Previous

### DISCUSSION THREAD

**Discussion by DustinVG** [MikeMaximus/gbm#284] [Ideas]
Basically, if the checksums of the two backups (sans configuration file) match, delete the new backup in order to save a bit of space and keep things more organized, since it's not actually needed.

Not sure how doable this is (it would require a check _before_ whenever GBM inserts said config file and a cache of the previous checksum for each game) and it might not even be worth it, but it's a thought I had.

**Comment by MikeMaximus** [comment]
Thanks for the suggestion.  I'm not sure if it's feasible to implement but I'll ponder it.
